testgrid/config.yaml

```diff
@@ -3707,6 +3709,13 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 8
+    num_failures_to_alert: 4 # Runs every 2h. Alert when it's been
+    # failing for 8 hours
+  - name: kubeadm-kinder-external-etcd-master
+    test_group_name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-master
+    alert_options:
+      alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
+    alert_stale_results_hours: 8
```

Comment:
if we change the rate from 2h, `num_failures_to_alert` and `alert_stale_results_hours` would need a tweak too. [1]

Reply:
As commented, leaving as is for now.